templates/download/iot/index.html

```diff
@@ -8,100 +8,77 @@
 <section class="p-strip is-bordered">
   <div class="row">
     <h1>Get Ubuntu for IoT</h1>
-    <p>Ubuntu is supported on a variety of chipsets, boards, SOCs and virtual machines, including Raspberry Pi 2 and 3, Compute Module 3, Qualcomm DragonBoard 410c, Intel NUC, Intel Joule, Samsung Artik and KVM.</p>
-    <p>We will guide you through the steps of installing Ubuntu Core, Ubuntu Desktop, or Ubuntu Server on these platforms.</p>
+    <p>Ubuntu Core is a tiny, transactional version of Ubuntu for IoT devices and large container deployments. It runs a new breed of super-secure, remotely upgradeable Linux app packages known as snaps ‐it’s trusted by leading IoT players, from chipset vendors to device makers and system integrators.</p>
+    <p><a href="/core">Learn more about Ubuntu Core ›</p><p> <a href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/" class="p-link--external">Browse all supported images</a></p>
```

Comment:
You shouldn't put two paragraphs on one line.

```suggestion
    <p><a href="/core">Learn more about Ubuntu Core ›</p>
    <p><a href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/" class="p-link--external">Browse all supported images</a></p>
```